tutoraspects/plugin.py

```diff
@@ -355,7 +355,7 @@
         # For now we are pulling this from github, which should allow maximum
         # flexibility for forking, running branches, specific versions, etc.
         ("DBT_REPOSITORY", "https://github.com/openedx/aspects-dbt"),
-        ("DBT_BRANCH", "v3.28.1"),
+        ("DBT_BRANCH", "bmtcril/update_instance_mvs"),
```

Comment:
Needs to be changed before merge: https://github.com/openedx/aspects-dbt/pull/98